- Docker image should be in the docker directory (see master branch)
- bin directory should not be under version control
- Docker image for npn fails with the following message:

```
web_1  | npm info lifecycle yanb@0.0.1~prestart: yanb@0.0.1
web_1  | npm info lifecycle yanb@0.0.1~start: yanb@0.0.1
web_1  | 
web_1  | > yanb@0.0.1 start /src
web_1  | > gulp --gulpfile gulpfile.js
web_1  | 
web_1  | [10:51:21] Local gulp not found in /src
web_1  | [10:51:21] Try running: npm install gulp
web_1  | 
web_1  | npm info lifecycle yanb@0.0.1~start: Failed to exec start script
web_1  | npm ERR! Linux 4.4.0-28-generic
web_1  | npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
web_1  | npm ERR! node v6.3.0
web_1  | npm ERR! npm  v3.10.3
web_1  | npm ERR! code ELIFECYCLE
web_1  | npm ERR! yanb@0.0.1 start: `gulp --gulpfile gulpfile.js`
web_1  | npm ERR! Exit status 1
web_1  | npm ERR! 
web_1  | npm ERR! Failed at the yanb@0.0.1 start script 'gulp --gulpfile gulpfile.js'.
web_1  | npm ERR! Make sure you have the latest version of node.js and npm installed.
web_1  | npm ERR! If you do, this is most likely a problem with the yanb package,
web_1  | npm ERR! not with npm itself.
web_1  | npm ERR! Tell the author that this fails on your system:
web_1  | npm ERR!     gulp --gulpfile gulpfile.js
web_1  | npm ERR! You can get information on how to open an issue for this project with:
web_1  | npm ERR!     npm bugs yanb
web_1  | npm ERR! Or if that isn't available, you can get their info via:
web_1  | npm ERR!     npm owner ls yanb
web_1  | npm ERR! There is likely additional logging output above.
web_1  | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
web_1  | 
web_1  | npm ERR! Please include the following file with any support request:
web_1  | npm ERR!     /src/npm-debug.log
yanb_web_1 exited with code 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,17 +1,14 @@
-FROM node:6.3.0-slim
+FROM node:6.2.1-slim
 
 MAINTAINER ivanvs
 
 RUN apt-get update -qq && apt-get install -y build-essential
 
-RUN apt-get install -y ruby
-RUN gem install sass
-
 RUN mkdir /src
+WORKDIR /src
 
 RUN npm install gulp -g
 
-WORKDIR /src
 ADD package.json /src/package.json
 RUN npm install
 

--- a/bin/www
+++ b/bin/www
@@ -1,9 +1,8 @@
 #!/usr/bin/env node
-var debug = require('debug')('server');
 var app = require('../server');
 
 app.set('port', process.env.PORT || 3000);
 
 var server = app.listen(app.get('port'), function() {
-  debug('Express server listening on port ' + server.address().port);
+  console.log('Express server listening on port ' + server.address().port);
 });

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -13,7 +13,8 @@ services:
   environment:
    - NODE_ENV=development
   volumes:
-   - ".:/src"
+   - .:/src
+   - /src/node_modules
   depends_on:
     - db
   links:

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -30,7 +30,7 @@ gulp.task('watch', function () {
     gulp.watch('./**/*.js', ['lint']);
 });
 
-gulp.task('develop', ['lint', 'test'], function () {
+gulp.task('develop', function () {
   nodemon({script: './bin/www', ext: 'js', legacyWatch: true });
 });
 

